api/src/main/java/org/eclipse/microprofile/metrics/SimpleTimer.java

```diff
@@ -98,5 +98,33 @@
 
     @Override
     long getCount();
+    
+    /**
+     * Get the maximum recorded time duration of the SimpleTimer for the previously completed full minute.
+     * <p>
+     * This represents the highest timed duration in the last completed full minute. For example if the last completed
+     * full minute recorded durations of 12 seconds, 9 seconds and 1 second the largest value is 12 seconds.
+     * 
+     * If no time duration has been recorded in the last completed full minute then a null is returned.
+     * If there has only been one time duration recorded in the last completed full minute then this value will be returned
+     * by both {@link SimpleTimer#getMaxTimeDuration()} and {@link SimpleTimer#getMinTimeDuration()}
+     *
+     * @return The maximum recorded time duration in the previously completed full minute.
```

Comment:
Update to say that it returns null if no measurements are available

Reply:
Added to the @return statement now as well.